ssh-notes/ssh-connection-problems.md

```diff
@@ -0,0 +1,138 @@
+# Connection Problems
+
+This is intended to address a very specific problem you may encounter when
+
+- trying to use SSH
+- from a WSL 2 distrubution
+- running under Windows
+
+## How to Tell if you have the problem
+
+The most obvious symptom is that when you try and perform any SSH related activities, the SSH program appears to hang.
+
+This problem will occur with any SSH command that has to try and send a key across the network, so it could affect connections to any SSH server (e.g. Azure).
+
+To check if you are having the specific connection problem this article attempts to address, do the following:
+
+1. Start a WSL shell
+2. Run the following command
+  ```bash
+  ssh -T -vv git@github.com
+  ```
```

Comment:
```suggestion

   ```text
   ssh -T -vv git@github.com
   ```

```

Reason: Indentation does not match prior text start.

Reason: syntax highlighting sometimes "disappears". Using `text` is cleaner in this instance.

Reason: MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines (markdownlint)